include/deal.II/base/exceptions.h

```diff
@@ -18,7 +18,9 @@
 
 #include <deal.II/base/config.h>
 
-#include <deal.II/base/kokkos.h>
+DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
+#include <Kokkos_Core.hpp>
+DEAL_II_ENABLE_EXTRA_DIAGNOSTICS
```

Comment:
If you rebase, you can probably remove the extra diagnostics thingt.

Reply:
Fixed.